tests/dist/chats.spec.dev.js

```diff
@@ -229,58 +237,59 @@ describe('chat resolvers', function () {
       createGroupChatResponse,
       createGroupChatData;
     return regeneratorRuntime.async(function _callee5$(_context5) {
-      while (1) {
+      const condition = 1;
+      while (condition) {
         switch ((_context5.prev = _context5.next)) {
           case 0:
             nameForNewUser = shortid.generate();
             email = ''.concat(nameForNewUser, '@test.com');
             _context5.next = 4;
             return regeneratorRuntime.awrap(
-            axios.post(URL, {
-              query: '\n              mutation {\n                  signUp(data: {\n                  firstName:"'
-                .concat(nameForNewUser, '",\n                  lastName:"')
-                .concat(nameForNewUser, '"\n                  email: "')
-                .concat(
-                  email,
-                  '"\n                  password:"password"\n                  }) {\n                  user{\n                    _id\n                  }\n                  accessToken\n                  }\n              }\n              '
-                ),
-            })
-          );
+              axios.post(URL, {
+                query: '\n              mutation {\n                  signUp(data: {\n                  firstName:"'
+                  .concat(nameForNewUser, '",\n                  lastName:"')
+                  .concat(nameForNewUser, '"\n                  email: "')
+                  .concat(
+                    email,
+                    '"\n                  password:"password"\n                  }) {\n                  user{\n                    _id\n                  }\n                  accessToken\n                  }\n              }\n              '
+                  ),
+              })
+            );
```

Comment:
It would be ideal to use a template literal here to improve the readability of this query.